### PR TITLE
[18.0][MIG] mgmtsystem_survey: Migration to 18.0

### PR DESCRIPTION
**Pull request by Kimkhoi3010**

### CODE REVIEW

**Review by henrybackman — APPROVED** [review]

File: mgmtsystem_survey/readme/CREDITS.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+The development of this module has been financially supported by:
+
+- Camptocamp
+
+The migration of this module from 17.0 to 18.0 was financially supported by:
+
+- Camptocamp.
```
Comment (ajaniszewska-dev) [inline mgmtsystem_survey/readme/CREDITS.md]:
```suggestion
- Camptocamp
```

**Review by ajaniszewska-dev — APPROVED** [review]

File: mgmtsystem_survey/readme/CREDITS.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+The development of this module has been financially supported by:
+
+- Camptocamp
+
```
Comment (max3903) [inline mgmtsystem_survey/readme/CREDITS.md]:
```suggestion
```

File: mgmtsystem_survey/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+{
+    "name": "Management System - Survey",
+    "version": "18.0.1.0.0",
+    "author": "Savoir-faire Linux, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/management-system",
+    "license": "AGPL-3",
+    "category": "Management System",
+    "depends": ["mgmtsystem", "survey"],
+    "data": ["views/survey_survey.xml"],
+    "installable": True,
```
Comment (max3903) [inline mgmtsystem_survey/__manifest__.py:13]:
```suggestion
    "installable": True,
    "development_status": "Beta",
    "maintainers": ["max3903"],
```

File: mgmtsystem_survey/models/__init__.py
Comment (max3903) [inline mgmtsystem_survey/models/__init__.py:1]: @Kimkhoi3010 Can you remove this file and its directory?

File: mgmtsystem_survey/__init__.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import models
```
Comment (max3903) [inline mgmtsystem_survey/__init__.py]:
```suggestion
```

File: mgmtsystem_survey/static/description/icon.png
Comment (max3903) [inline mgmtsystem_survey/static/description/icon.png:1]: @Kimkhoi3010 Can you update the icon please?

**Review by max3903 — CHANGES_REQUESTED** [review]
Minor changes. Thank you @Kimkhoi3010 !

File: mgmtsystem_survey/models/__init__.py
Comment (Kimkhoi3010) [reply mgmtsystem_survey/models/__init__.py:1]: Thanks for your time.

File: mgmtsystem_survey/static/description/icon.png
Comment (max3903) [reply mgmtsystem_survey/static/description/icon.png:1]: https://github.com/OCA/management-system/blob/18.0/mgmtsystem/static/description/icon.png

**Review by max3903 — APPROVED** [review]

**Pull request merged**